mozilla/fx-private-relay
# Add `store_phone_log` to `Profile` mpp 2250

This PR fixes MPP-2250.

How to test:
1. Check out this code
2. Run `python manage.py migrate`
3. Run `python manage.py runserver`
4. Sign in to http://127.0.0.1:8000
5. Go to http://127.0.0.1:8000/api/v1/profiles/
   * [x] there should be a `store_phone_log` item (defaults to `true`)
   * Note the `id` value
6. Go to http://127.0.0.1:8000/api/v1/docs/
7. Scroll down to the "profiles" endpoints
8. Expand the `PATCH /profiles/{id}/` item
9. Click "Try it out"
10. Change `  "store_phone_log": false,`
11. Enter the `id` value from step 5
12. Click "Execute"
13. Refresh http://127.0.0.1:8000/api/v1/profiles/
    * [x] The `store_phone_log` item should now show `false`
